### ISSUE TITLE
Slow-Fast Pytorch Finetuning

### ISSUE THREAD

**Issue by ooguz166** [dmlc/gluon-cv#1681]
Hi, I was trying to finetune the  pre-trained (in torch) "slowfast_4x16_resnet50_kinetics400" network and able to achieve:
 _"Top-1 accuracy: 88.51%, Top-5 accuracy: 99.23%"_ 
in validation split of UCF101 dataset in the end of the 2nd Epoch (which is reasonable since it is a pretrained model, and configurations are taken from slowfast_4x16_resnet50_custom.yaml)

However, I used the "build_dataloader" and VideoClsDataset in dataset_classification.py. I noticed that in mxnet version  we have to indicate that the net is a slow-fast network. In torch, however no such indication is made.

So should I need to use such dataloader ?

**Comment by ooguz166** [comment]
As I search through the slow fast PyTorch implementation  

        fast_input = x

        slow_input = x[:, :, ::self.slow_temporal_stride // 2, :, :]` is implemented in  SlowFast(nn.Module). Thus giving the required clip size seems to be ok. closing the issue.

**Comment by AndyDrill** [comment]
Hello @ooguz166 
May I ask some questions for training the SlowFast Pytorch Implementation?
